crates/test-cli/src/lib.rs

```diff
@@ -132,6 +133,8 @@ enum CliCommand {
         config_interface_file: Option<PathBuf>,
         /// The path to a file containing the program aux interface (defaults to empty)
         aux_data_interface_file: Option<PathBuf>,
+        /// The path to a file containing the program oracle data (defaults to empty)
```

Comment:
```suggestion
        /// The path to a file containing the program oracle data headings (defaults to empty)
```
or similar - just to make clear that its the headings that this file contains because otherwise when i read i think of the actual data that the oracle retrieves